apps/api/src/migration/1692040928087-RenameWithdrawStatus.ts

```diff
@@ -0,0 +1,17 @@
+import { IENApplicantStatus } from 'src/applicant/entity/ienapplicant-status.entity';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RenameWithdrawStatus1692040928087 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.connection
+      .createQueryBuilder()
+      .update(IENApplicantStatus)
+      .set({
+        status: 'Not Proceeding',
```

Comment:
Don't we need to change `STATUS.WITHDREW_FROM_PROGRAM` too? It is also referenced in many places.

Reply:
I changed the value of WITHDREW_FROM_IEN_PROGRAM, but not the key. I have mixed feelings about doing that as it's a very large change.

Reply:
I got it, but at least doesn't STATUS need to be updated like this?
`WITHDREW_FROM_PROGRAM = 'Not Proceeding'`